libs/damap/src/lib/widgets/dmp-table/dmp-table.component.html

```diff
@@ -1,118 +1,126 @@
-<mat-form-field>
-  <mat-label>{{ "plans.table.search" | translate }}</mat-label>
-  <input
-    matInput
-    (keyup)="applyFilter($event)"
-    placeholder="{{ 'plans.table.search' | translate }}"
-    #input />
-</mat-form-field>
+<div class="button-and-search">
+  <button class="button-color-primary" [routerLink]="'/dmp'" mat-raised-button>
+    <mat-icon>add</mat-icon>
+    {{ "plans.create.dmp" | translate }}
+  </button>
 
-<table
-  mat-table
-  [dataSource]="dataSource"
-  multiTemplateDataRows="true"
-  class="mat-elevation-z8"
-  style="outline: none"
-  matSort
-  aria-label="Available DMPs">
-  <ng-container matColumnDef="title">
-    <th mat-header-cell *matHeaderCellDef mat-sort-header>
-      {{ "plans.table.header.title" | translate }}
-    </th>
-    <td mat-cell *matCellDef="let element">
-      <a class="title" routerLink="/dmp/{{ element.id }}">
-        <h3 matTooltip="Edit DMP">
-          {{ element.project?.title || "DMP ID: " + element.id }}
-        </h3>
-      </a>
-    </td>
-  </ng-container>
+  <damap-search-field
+    *ngIf="(dmpsLoaded | async) === LoadingState.LOADED"
+    class="search-bar"
+    [label]="'plans.table.search'"
+    [appearance]="'outline'"
```

Comment:
minor: outline is the default - can be omitted here (probably other occasions as well)